.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,17 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
 
 I modified the labyrinth, you can test my changes here:
-http://rawgit.com/USERNAME/labyrinth/BRANCH-NAME/index.html
-<!-- Please replace USERNAME with your GitHub user name and
-     BRANCH-NAME with your branch name e.g. master -->
+ http://rawgit.com/USERNAME/labyrinth/BRANCH-NAME/index.html
+ <!-- Please replace USERNAME with your GitHub user name and
+      BRANCH-NAME with your branch name e.g. master -->
+#### Checklist
 
+- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
```

Comment:
I do not know if that is really necessary and adds a lot of comlexity for new-comers.